src/pkg/packaging/dir.proj

```diff
@@ -154,6 +154,8 @@
   </PropertyGroup>
   <Target Name="GenerateInstallers" DependsOnTargets="InitPackage;$(InstallerDependsOn)" />
 
+  <Import Project="$(ProjectDir)src/managed/CommonManaged.props" />
```

Comment:
CommonManaged.props sets the `Serviceable` and `IncludeSymbols` properties used for packaging